lighthouse-core/test/gather/driver-test.js

```diff
@@ -876,24 +905,27 @@ describe('Domain.enable/disable State', () => {
     connectionStub.sendCommand = createMockSendCommandFn()
       .mockResponse('Network.enable')
       .mockResponse('Network.disable')
+      // @ts-ignore: Fetch is not in protocol typings yet.
       .mockResponse('Fetch.enable')
       .mockResponse('Fetch.disable');
 
-    await driver.sendCommand('Network.enable', {});
-    await driver.sendCommand('Network.enable', {});
+    await driver.sendCommand('Network.enable');
+    await driver.sendCommand('Network.enable');
     expect(connectionStub.sendCommand).toHaveBeenCalledTimes(1);
 
-    await driver.sendCommand('Network.disable', {});
+    await driver.sendCommand('Network.disable');
     expect(connectionStub.sendCommand).toHaveBeenCalledTimes(1);
     // Network still has one enable.
 
+    // @ts-ignore: Fetch is not in protocol typings yet.
```

Comment:
can/should we update? or it's not even typed at all yet?

Reply:
was gonna defer to #9459 but we can just update now